Sources/HipMobileUI/ViewModels/Views/ForgotPasswordScreenViewModel.cs

```diff
@@ -46,6 +46,10 @@ private void OnResetPasswordClicked()
             {
                 DisplayEmptyEmailErrorMessage();
             }
+            else if (!(Email.Contains("@") && Email.Contains(".")))
```

Comment:
This is incorrect, you should only check for an `@`. [See why](https://stackoverflow.com/a/20573804/1502352). You should also change `RegisterScreenViewModel.OnRegisterClicked` accordingly.